frontend/coprs_frontend/coprs/forms.py

```diff
@@ -9,19 +9,21 @@
 from flask_wtf.file import FileRequired, FileField
 from fnmatch import fnmatch
 
+from coprs.constants import DEFAULT_REPO_PRIORITY_VALUE
```

Comment:
I think if you move this import closer to the rest of `coprs` imports, you will fix both linter errors.